front/src/pages/settings/SettingsWorkspaceMembers.tsx

```diff
@@ -74,6 +92,29 @@ export function SettingsWorkspaceMembers() {
     <NoTopBarContainer>
       <StyledContainer>
         <MainSectionTitle>Members</MainSectionTitle>
+        {workspace?.inviteHash && (
```

Comment:
Can we put this into a separate component ?

Reply:
Done here: https://github.com/twentyhq/twenty/pull/603/commits/f18099f2cb39e216925997f4aa8512f14d9455b0